data-analytics/dataflow-python-examples/google/data_ingestion.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""data_ingestion.py is a Dataflow pipeline which reads a file and writes its 
+contents to a BigQuery table.
+This example does not do any transformation on the data. 
+ 
+"""
+
+from __future__ import absolute_import
+import argparse
+import logging
+import re
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+class DataIngestion:
+    def __init__(self):
+        """A helper class which contains the logic to translate the file into 
+        a format BigQuery will accept"""
+
+    @staticmethod
```

Comment:
avoid use of @staticmethod, https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#217-function-and-method-decorators